anta/tests/interfaces.py

```diff
@@ -964,45 +964,80 @@ def validate_interfaces(cls, interfaces: list[T]) -> list[T]:
                     raise ValueError(msg)
             return interfaces
 
+    def _verify_port_channel_intf_churn_state(self, interface_details: dict[str, Any], interface: str) -> None:
```

Comment:
Type annotation of interface should be `InterfaceState` no?

Also, we should put the interface as the first parameter of these functions for clarity. We should also probably rename the parameters:
```suggestion
    def _verify_port_channel_intf_churn_state(self, interface_input: InterfaceState, interface_output_data: dict[str, Any], interface: str) -> None:
```

Let's also reduce the lenght of the function name... Apologies for changing my mind very often :D 

`_verify_interface_churn_state` is better IMO.